sql/service_wsrep.cc

```diff
@@ -29,14 +29,12 @@ extern "C" my_bool wsrep_on(const THD *thd)
 
 extern "C" void wsrep_thd_LOCK(const THD *thd)
 {
-  mysql_mutex_lock(&thd->LOCK_thd_kill);
```

Comment:
Splitting `LOCK_thd_kill` and `LOCK_thd_data` to separate calls to allow BF aborting of SR transaction from InnoDB lock manager. BF aborting SR transaction grabs `LOCK_thd_kill` in `THD::reset_globals()` which causes
`LOCK_thd_kill` to be locked twice if `wsrep_thd_LOCK()` locks it too.